### PR TITLE
Samples reorganized

### PR DESCRIPTION
**Pull request by HemantMahawar**

### CODE REVIEW

File: docs/PublicAPIsAndMore.md [hunk 1]
```diff
@@ -70,41 +67,40 @@ SHiPS contains the following attributes:
...
   As above mentioned, an author can define dynamic parameters for their providers.
   To process whether a user uses these dyanmic parameters, author can access them via the following mechanism.
 
-    ``` PowerShell
+    ```powershell
     $this.ProviderContext.DynamicParameters
-
     ```
 
     For more usages, see [dynamic parameters  example][ds].
```
Comment (maertendMSFT) [inline docs/PublicAPIsAndMore.md]: The "dynamic parameters example" link is broken

File: docs/Design.md [hunk 2]
```diff
@@ -86,50 +86,42 @@ Mode       Name
...
 
-
-</br>
-
-![pie](pie.png)
-
-
-</br>
-
+![pie](images/pie.png)
```
Comment (maertendMSFT) [inline docs/Design.md]: Cannot find the image

File: docs/README.md [hunk 3]
```diff
@@ -77,12 +84,13 @@
   ```
 
 ## Step 3 - Use MyProvider.psm1
-- Import-Module SHiPS (See [`Installing SHiPS` ][readme] for details)
+
+- Import-Module SHiPS (See [`Installing SHiPS`][readme] for details)
```
Comment (maertendMSFT) [inline docs/README.md]: Should "Installing SHiPS" be inside `` ?

File: docs/PublicAPIsAndMore.md (hunk 1 above)
Comment (HemantMahawar) [reply docs/PublicAPIsAndMore.md]: fixed. Will update the PR

File: docs/Design.md (hunk 2 above)
Comment (HemantMahawar) [reply docs/Design.md]: Fixed. Will update the PR

File: samples/FamilyTreeInCSharp/README.md [hunk 4]
```diff
@@ -1,25 +1,22 @@
-
-# Sample - FamilyTree Written in C# #
-
+# Sample Provider - FamilyTree Written in C# #
 
 As you may have tried the [FamilyTree][ft] written in PowerShell class.
```
Comment (maertendMSFT) [inline samples/FamilyTreeInCSharp/README.md]: Broken "FamilyTree" link

Comment (HemantMahawar) [reply samples/FamilyTreeInCSharp/README.md]: fixed. Will update the PR.

**Comment by maertendMSFT** [comment]
@HemantMahawar, I commented on everything I found.  I am done going through the docs

File: docs/README.md (hunk 3 above)
Comment (HemantMahawar) [reply docs/README.md]: fixed. Will update the PR.

**Pull request merged**